src/config_ctrl.ts

```diff
@@ -6,7 +6,8 @@ export class IrondbConfigCtrl {
 
   /** @ngInject */
   constructor($scope) {
-    this.current.jsonData.irondbType = this.current.jsonData.irondbType || 'standalone';
+    this.current.jsonData.irondbType = this.current.jsonData.irondbType || 'hosted';
```

Comment:
Was this change to the default intended?

Reply:
yes, but I don't remember if I had asked...should 'standalone' be the default instead?